### DISCUSSION TITLE
How do I credit discord.js in a separate project?

### DISCUSSION THREAD

**Discussion by lewisfm** [discordjs/discord.js#8554] [Q&A] [answered]
I really like how the discord.js contributors have set up the GitHub issue templates and I'm going to use a similar format in my own project, which uses the ISC license rather than discord.js's Apache license. While the issue template is highly modified so as to fit my use case, some phrases remain the same. I believe this makes it similar enough to be considered a modification rather than simply taking inspiration. What is the correct way to credit the discord.js contributors in this situation? Should I simply add an acknowledgement at the top of the file? Or do I need to add a 2nd license to my project?

**Comment by almostSouji** [comment, marked as answer]
I don't think wording choices or template setups fall under any sort of license. I wrote them based on what I saw need for in our repository and we re-iterated them as we saw the need arise.

If you still want to credit us, feel free to do so as you see fit. Just drop the link to the repository into a comment or readme, for example.
